files/galaxy/tpv/tools.yml

```
@@ -1223,7 +1223,19 @@ tools:
       require:
         - singularity
 
+  braker_tool:
+    abstract: true
+    rules:
+      - id: braker_genome_sequence_length_rule
```

its not sequence length, its sequence count 

```suggestion
      - id: braker_genome_sequence_count_rule
```

---

Good catch! Thanks!